MovingBlocks/Terasology
# Improve Simplex and Perlin noise generators

### Contains

More control on Simplex and Perlin noise generators, Adds Docs to use them as Tileable noise generators

### How to test

Run these two branches of the module I made for testing, use the world generator by the name of "oki" in game
https://github.com/sin3point14/WorldGenTesting/tree/simplex-test
https://github.com/sin3point14/WorldGenTesting/tree/perlin-test

You should see flanked mountains like this
![image](https://user-images.githubusercontent.com/43549821/85226002-a6326700-b3f2-11ea-98d4-3792ca43bfe8.png)
the "flankiness" will be controllable by changing `SurfaceProvider.gridSize` in the module
